cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/RelationalTranformContext.java

```diff
@@ -66,4 +69,12 @@ public interface RelationalTranformContext extends FeatureFlagsProvider {
    *
    */
   void setOutputRelation(String portName, Relation outputDataSet);
+
+  /**
+   * Gets the list of languages that the relation transform context supports by default.
+   * @return the list of languages supported by default as a list of {@link Capability}s.
+   */
+  default Collection<Capability> getDefaultLanguageCapabilityList() {
+    return Collections.emptySet();
```

Comment:
Please `throw UnsupportedOperationException`. It's not expected that anyone but platform needs to implement this.